include/user_functions/TGMMSElemKernel.h

```diff
@@ -0,0 +1,51 @@
+  /*------------------------------------------------------------------------*/
+/*  Copyright 2014 Sandia Corporation.                                    */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#ifndef TGMMSElemKernel_h
+#define TGMMSElemKernel_h
+
+#include <kernel/Kernel.h>
+#include <FieldTypeDef.h>
+#include <AlgTraits.h>
+
+#include <stk_mesh/base/BulkData.hpp>
+#include <stk_mesh/base/Entity.hpp>
+#include <memory>
+
+namespace sierra{
+namespace nalu{
+
+class Realm;
+class ElemDataRequests;
+
+template <class AlgTraits>
+class TGMMSElemKernel final : public Kernel
```

Comment:
Can you add a comment explaining TGMMS? Should there be an addition to verification section of the nalu docs?

Reply:
Added a comment.  It's a simplified version of the Boussinesq MMS case.  It's nice for development IMO but it's just verifying a subset of physics that's already verified by a number of other verification tests, so I wasn't planning on doing separate documentation for it.